src/H5FDsubfile_mpi.c

```diff
@@ -31,8 +31,8 @@ static double sf_queue_delay_time = 0.0;
  * intend to use the user defined HDF5 filename for a
  * zeroth subfile as well as for all metadata.
  */
-#define SF_FILENAME_TEMPLATE ".subfile_%ld_%d_of_%d"
-
+//#define SF_FILENAME_TEMP_TEMPLATE ".subfile_%%ld_%%0%dd_of_%%d"
```

Comment:
Probably don't want to check in commented-out code

Reply:
fixed